modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java

```diff
@@ -446,6 +449,14 @@ public void execute(Throwable throwable) {
 
 			});
 
+		DockerPruneImage dockerPruneImage = GradleUtil.addTask(
+			project, PRUNE_DOCKER_IMAGE_TASK_NAME, DockerPruneImage.class);
+
+		dockerPruneImage.setDescription("Prunes the Docker image.");
+		dockerPruneImage.setGroup(DOCKER_GROUP);
+
+		dockerBuildImage.finalizedBy(PRUNE_DOCKER_IMAGE_TASK_NAME);
```

Comment:
Hey @Seiphon  can we remove this?  I don't think we should "prune" on each "build".  That is too much churn.  With ehse new commands, developer can "prune" when they need.

Reply:
OK，got it, I will remove them